Sources/XcodeProj/Objects/SwiftPackage/XCSwiftPackageProductDependency.swift

```diff
@@ -37,6 +41,11 @@ public class XCSwiftPackageProductDependency: PBXContainerItem, PlistSerializabl
         } else {
             packageReference = nil
         }
+        if let plugin = try container.decodeIfPresent(Bool.self, forKey: .plugin) {
+            self.plugin = plugin
+        } else {
+            plugin = false
+        }
```

Comment:
if I followed along, there isn't a `plugin` attribute in the pbxproj file, rather the product name is prefixed with `plugin:` - in that case would this not require checking for that instead?

Reply:
@kwridan Good point! I just realized that I completely misunderstood what this initializer is for. My bad 🤦‍♂️

Reply:
no worries